randovania/data/json_data/prime2.txt

```diff
@@ -3364,6 +3368,8 @@ Asset id: 3212793644
 
 > Event - Portal Site Cutscene; Heals? False
   * Event Portal Site Cutscene
+  > Portal to Portal Terminal
+      Knowledge (Beginner)
   > Safe Zone (Bomb Slot)
       Trivial
```

Comment:
Accessing the portal from the cutscene is already in logic, so this isn't needed. The only connection this node should have is to the Portal as Trivial.